### PR TITLE
Fix conditional hook

### PR DESCRIPTION
**Pull request by tdraier**

### CODE REVIEW

File: front/.eslintrc.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ module.exports = {
     "@typescript-eslint/no-explicit-any": 0,
     "@typescript-eslint/no-unused-vars": "error",
     "no-case-declarations": 0,
-    "react-hooks/rules-of-hooks": 0,
+    "react-hooks/rules-of-hooks": "error",
```
Comment (lasryaric) [inline front/.eslintrc.js:26]: will set it in a separate PR

**Review by lasryaric — COMMENTED** [review]
LGTM minus the change to the eslint settings.

File: front/.eslintrc.js (hunk 1 above)
Comment (tdraier) [reply front/.eslintrc.js]: commit removed

**Review by lasryaric — APPROVED** [review]
LGTM

**Pull request merged**